src/scripts/settings/userSettings.ts

```diff
@@ -421,25 +482,29 @@ export class UserSettings {
      * @param {number|undefined} val - The interval between backdrops in seconds.
      * @return {number} The interval between backdrops in seconds.
      */
-    backdropScreensaverInterval(val) {
+    backdropScreensaverInterval(): number;
+    backdropScreensaverInterval(val: number): void;
+    backdropScreensaverInterval(val?: number): number | void {
         if (val !== undefined) {
             return this.set('backdropScreensaverInterval', val.toString(), false);
         }
 
-        return parseInt(this.get('backdropScreensaverInterval', false), 10) || 5;
+        return parseInt(this.get('backdropScreensaverInterval', false) || '5', 10);
     }
 
     /**
      * Get or set library page size.
      * @param {number|undefined} val - Library page size.
      * @return {number} Library page size.
      */
-    libraryPageSize(val) {
+    libraryPageSize(): number;
+    libraryPageSize(val: number): void;
+    libraryPageSize(val?: number): number | void {
         if (val !== undefined) {
             return this.set('libraryPageSize', val.toString(), false);
         }
 
-        const libraryPageSize = parseInt(this.get('libraryPageSize', false), 10);
+        const libraryPageSize = parseInt(this.get('libraryPageSize', false) || '', 10);
```

Comment:
Should this be `'0'` like the next instance?

```suggestion
        const libraryPageSize = parseInt(this.get('libraryPageSize', false) || '0', 10);
```

Reply:
With `'0'`, we won't get the default `100`.